packages/app/tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
-  "exclude": ["script/generatePreload.ts"],
+  "exclude": ["script/generatePreload.ts", "dist"],
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
     "module": "commonjs",
     "sourceMap": true,
     "strict": true,
     "outDir": "./dist",
     "rootDir": "./src",
+    "skipLibCheck": true,
```

Comment:
reminder: check if skipLibCheck is still required after fixing some of the outboard libraries